README.md

```diff
@@ -1,2 +1,2 @@
 # quiz-creator
-An interface to create quizzes for the Avanti Fellows Quiz Engine
+An interface to create quizzes for the Avanti Fellows Quiz Engine and Remove the dependence on google sheets completely
```

Comment:
, and remove dependency on Google Sheets completely.

Reply:
Updated